Awful.apk/src/main/java/com/ferg/awfulapp/widget/AlertView.kt

```diff
@@ -17,12 +16,24 @@ import com.ferg.awfulapp.util.show
 import timber.log.Timber
 
 
+/**
+ * The purpose of the clear in [displayAlert] is to clear all the inputs to reset so the next call will be a fresh start
```

Comment:
Can you move the ``displayAlert`` explanation to the docstring for that method? Like a one-liner saying that it pops up the toast with the params you've set, and then you can go into detail with it

And just add an explanation of what this class is in the class docstring (displays a custom toast with optional subtitle and icon, uses a builder pattern and call ``#show`` to display it). And yeah leave the TODO in here, I think it's the best place for it. Can you stick the url in a link so it's clickable in the doc popup? Like
``[this link](https://stackoverflow.com/questions/47016590/stringres-drawableres-layoutres-and-so-on-android-annotations-lint-check-wi)``
or whatever text you like. Cheers!